R/mlClusteringDensityBased.R

```diff
@@ -167,18 +167,21 @@ mlClusteringDensityBased <- function(jaspResults, dataset, options, ...) {
     suggestedLine <- NULL
   else
     suggestedLine <- data.frame(xstart = xBreaks[1], xend = xBreaks[length(xBreaks)], ystart = yKnee, yend = yKnee)
-  
+
   lineData <- data.frame(xstart = xBreaks[1], xend = xBreaks[length(xBreaks)], ystart = options[["eps"]], yend = options[["eps"]])
- 
-  p <-  ggplot2::ggplot(data = d, ggplot2::aes(x = x, y = y)) + 
+
+  p <-  ggplot2::ggplot(data = d, ggplot2::aes(x = x, y = y)) +
         ggplot2::scale_x_continuous(name = gettext("Points sorted by distance"), breaks = xBreaks) +
         ggplot2::scale_y_continuous(name = gettextf('%s-nearest neighbors \ndistance', options[['minPts']]), breaks = yBreaks)
```

Comment:
Should we add `limits = range(yBreaks)`?

Without `limits = range(yBreaks)`:
![image](https://user-images.githubusercontent.com/25059399/116521956-e85eeb80-a8d4-11eb-9d96-6e7e6823ae81.png)

With `limits = range(yBreaks)`:
![image](https://user-images.githubusercontent.com/25059399/116522068-088eaa80-a8d5-11eb-87f1-876cfabf589f.png)

Reply:
good idea, I'll add it